src/solver/batchnorm/backward_ck.cpp

```diff
@@ -362,7 +362,7 @@ bool BnCKBwdBackward::IsApplicable(
         return false;
     if(!bn_problem.IsLayoutNHWC() && !bn_problem.IsLayoutNCHW())
         return false;
-    if(!ck_utility::is_ck_supported_hardware(context.GetStream()))
```

Comment:
`is_ck_supported_hardware` relates to the `Legacy CK`, while this solver uses `CK`.